plugins/modules/monitoring/newrelic_deployment.py

```diff
@@ -30,7 +30,7 @@
   application_id:
     type: str
     description:
-      - (one of app_name or application_id are required) The application id, found in the URL when viewing the application in RPM
+      - (one of app_name or application_id are required) The application ID, found in the metadata of the application in APM
```

Comment:
While you're at it:
```suggestion
      - The application ID, found in the metadata of the application in APM.
      - One of I(app_name) or I(application_id) is required.
```
(Similar change for the description of `app_name`.)